packages/terra-time-input/src/terra-dev-site/test/time-input/twelve-hour/DefaultSecond.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import TimeInput from '../../../../TimeInput';
+import TimeUtil from '../../../../TimeUtil';
+
+class TimeInputDefault extends React.Component {
```

Comment:
Can we switch these test components to be functional component instead of classes?

Reply:
Agree, though I'd also be fine with that being a separate PR if it's easier since these are functionally equiv. and the class is small.

Reply:
Swapped the new tests i added in to functional components because we do have some as other tests already.
https://github.com/cerner/terra-framework/pull/1503/commits/aef1fe13be57204c740370d8c4090c417fb8515d#diff-be533d9e7f632d061195deb6232f1b652fe3baadf9123dcc6aa91625ff06da31R5